introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    version = 1
</pre><p>I recommend using a timestamp suffix instead of hard-coded version number so a new value will be auto-generated each time the notebook is run. </p><p>Suggestion: </p><pre class="ql-syntax" spellcheck="false">import gmtime
timestamp_suffix = time.strftime("-%Y-%m-%d-%H-%M-%S", time.gmtime())
</pre><p>That will give you a unique string like: <code>-2022-08-08-15-29-18</code>  that you can append to any job or resource name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>